### ISSUE TITLE
QT UIs pollute git

### ISSUE THREAD

**Issue by rene-dev** [LinuxCNC/linuxcnc#2704]
The QT UIs seem to store some preferences in the sim folder.
after playing around with them, the git is dirty.
settings do not belong in a file tracked by version control.

```
modified:   configs/sim/qtplasmac/qtplasmac-metric.prefs
modified:   configs/sim/qtplasmac/qtvcp.prefs
```

**Comment by andypugh** [comment]
Add to .gitignore?

**Comment by rene-dev** [comment]
Gitignore would mean deleting the file. I don't know what the file does, or if it contains default values that are required.

**Comment by phillc54** [comment]
Some of the preferences in these files show some options available in the GUI.

**Comment by rene-dev** [comment]
if the files are deleted, the UI will recreate them. the config folder is the wrong place for a config file.

**Comment by andypugh** [comment]
I think that the user's private config folder is the right place for a config file. 
It's just a feature of the RIP install that leads to them being created there. 

If the GUIs recreate them if missing, then I think that .gitignore is the answer.

**Comment by c-morley** [comment]
git ignore is the answer - though I can't get it to work yet.
The preference file will end up in the user's config folder on a real machine.

**Comment by c-morley** [comment]
@phillc54 can the pref files be uncommitted then ignored. ie Qtplasmac will work fine with no pref files the first time run?

**Comment by phillc54** [comment]
Yes, there are reasonably sane defaults that it will self populate with.

**Comment by c-morley** [comment]
@phillc54 you want to do this or I can - just so we can close the issue.

**Comment by phillc54** [comment]
I will get to it either tomorrow or Monday.

**Comment by phillc54** [comment]
Closed with [d7eff7f](https://github.com/LinuxCNC/linuxcnc/commit/d7eff7fabd3b80eedcfc17568f761dba5060f70c)
